test/e2e/mock-e2e-allowlist.js

```diff
@@ -13,7 +13,6 @@ const ALLOWLISTED_URLS = [
   'https://bafkreifvhjdf6ve4jfv6qytqtux5nd4nwnelioeiqx5x2ez5yrgrzk7ypi.ipfs.dweb.link/',
   'https://bafybeidxfmwycgzcp4v2togflpqh2gnibuexjy4m4qqwxp7nh3jx5zlh4y.ipfs.dweb.link/1.json',
   'https://bridge.api.cx.metamask.io/getTokens?chainId=1',
-  'https://cdn.contentful.com/spaces/jdkgyfmyd9sw/environments/master/entries?content_type=productAnnouncement&order=-sys.createdAt&fields.clients=portfolio',
```

Comment:
Removing this real-world call, we now utilise mocks.

Reply:
great :heart_eyes:  ty for adding mocks for this :pray: